cypress/support/fragments/settings/acquisitionUnits/acquisitionUnits.js

```diff
@@ -96,10 +96,9 @@ export default {
     cy.expect(auPaneDetails.find(assignedUsersSection).exists());
     // //Need to wait,while data of Acquisition Unit will be loaded
     cy.wait(5000);
-    cy.do([
-      actionsButton.click(),
-      Button('Edit').click()
-    ]);
+    cy.do(actionsButton.click());
+    cy.wait(5000);
```

Comment:
please add an explanation why cy.wait is needed here

Reply:
Donde